services/execdb/postgresql/upgrader.go

```diff
@@ -282,8 +287,9 @@ CREATE TABLE t_blocks (
  ,f_state_root               BYTEA NOT NULL
  ,f_withdrawals_root         BYTEA
  ,f_timestamp                TIMESTAMPTZ NOT NULL
- ,f_total_difficulty         NUMERIC NOT NULL
+ ,f_total_difficulty         NUMERIC
```

Comment:
Given we've changed the column type on init, should there not be a corresponding upgrade function to do the same for existing tables? I can only see us adding `f_requests_hash` to this table.